### DISCUSSION TITLE
Plugin Support - Extensions for Hoppscotch

### DISCUSSION THREAD

**Discussion by emindeniz99** [hoppscotch/hoppscotch#4115] [Ideas]
A sidebar tab that has a special ui that is developed by the plugin developer to generate, request body etc. UI has some buttons clear body, generate inputs in json with dummy variables etc...
Like vs code extension for vs code.
extensions for hoppscotch.
What do you think about it?

**Comment by Govindu564** [comment]
i want to extention

**Comment by Govindu564** [comment]
i want to extension
